homeassistant/components/config/automation.py

```diff
@@ -107,3 +108,12 @@ async def websocket_automation_trace(hass, connection, msg):
         }
 
     connection.send_result(msg["id"], automation_traces)
+
+
+@websocket_api.websocket_command({vol.Required("type"): "automation/trace/list"})
+@websocket_api.async_response
+async def websocket_automation_trace_list(hass, connection, msg):
```

Comment:
This could be a callback instead of a coroutine function since we don't await inside.

Reply:
This is corrected in #47632